README.md

```diff
@@ -1,7 +1,40 @@
-# nulecule-library
-A set of [nulecule](https://github.com/projectatomic/nulecule)-ized applications that can be used for reference or as part of your applications
+# Nulecule library
 
-###Communication channels
+A set of [nulecule](https://github.com/projectatomic/nulecule)-ized applications that can be used for reference or as part of your deployment.
```

Comment:
I still think we need to add a definition/explanation here.  What about this:

This library contains a set of working examples of composite container applications.  The applications are specified using the Nulecule Specification.  This specification allows for reusable components and defines a full multi-container application.  You may use the applications for reference or, in some cases, deployment.